lib/fortune.js

```diff
@@ -252,6 +253,14 @@ Fortune.prototype._exposeResourceDefinitions = function() {
   });
 };
 
+Fortune.prototype.customType = function(name, schema, options, schemaCallback) {
```

Comment:
Corrected. Right now, it only accepts user-level schemas and make validations against them. On data level any complex objects are converted to `mongoose.Schema.Mixed` data type on adapter level (see here: https://github.com/flyvictor/fortune/blob/master/lib/adapters/mongodb.js#L68). 
Also, simpler one-scheme syntax follows the same look and feel of an ordinal fortune resource.